app/controllers/general.php

```diff
@@ -489,7 +489,7 @@ function router(App $utopia, Database $dbForConsole, SwooleRequest $swooleReques
         $host = $request->getHostname() ?? '';
         $mainDomain = App::getEnv('_APP_DOMAIN', '');
         // Only run Router when external domain
-        if ($host !== $mainDomain && $host !== 'localhost') {
+        if ($host !== $mainDomain && $host !== 'localhost' && $host !== 'appwrite') {
```

Comment:
Not sure we want to hard code this here... 🤔  

can we declare it as a constant in init.php?